index.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Youtube-clone</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://kit.fontawesome.com/668d372fa7.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+    <header>
+        <div class="logo-nav-bar">
+            <!--i class="fas fa-bars"></i-->
+            <img class="img-logo" src="img/téléchargement.png" alt="youtube logo">
+        </div>
+
+        <div class="search-nav-bar">
+            <form action="#">
+                <input type="text" placeholder="Search.." name="search">
+                <button class="nav-bar-search-button" type="submit"><i class="fa fa-search"></i></button>
+            </form>
+        </div>
+
+        <div class="nav-bar-right-buttons">
+            <button type="submit"><i class="fas fa-upload"></i></button>
+            <button class="sign-in-button">Sign in</button>
+        </div>
+    </header>
+
+    <div class="container">
```

Comment:
**[Optional]** This Html semantic is not in right order. The basic structure for a page is:
<header></header>
<main></main>
<footer></footer>